spec/query_spec.cr

```diff
@@ -177,6 +177,28 @@ describe LuckyRecord::Query do
       results.map(&.name).should eq [] of String
     end
   end
+
+  # how should I test these methods?
+  pending "#joins methods for associations" do
+    it "inner joins on belongs to" do
+      insert_a_post
+      post = Post::BaseQuery.new.first
+      insert_a_comment(post.id)
+
+      result = Comment::BaseQuery.new.join_posts.first # creates join_post at the moment
+      result.post.should eq post
```

Comment:
How should I test that this works. Would I need to add a `post` field to the `Comment` and load the post there?

Reply:
I think for now a simple test like this is good, but it should also test the generated SQL. Until there is support for querying associations it’ll be hard to test without testing the generated SQL directly.